.github/workflows/python_cron.yml

```diff
@@ -110,6 +119,12 @@ jobs:
         run: ci/scripts/util_checkout.sh
       - name: Free Up Disk Space
         run: ci/scripts/util_cleanup.sh
+      - name: Cache Docker Volumes
+        uses: actions/cache@v1
+        with:
+          path: .docker
+          key: ${{ matrix.cache }}-${{ hashFiles('cpp/**') }}
```

Comment:
`hashFiles('cpp/**', 'python/**')`?

Reply:
The build time is majored by the cpp implementation, so I thought it's better to have cache hits if the underlying C++ implementation has not changed.